src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.body {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap};
+  padding: #{$padding};
+
+  max-width: calendarWidth($cell-width, 7, $gap, $padding);
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    height: 100px;
+    width: #{$cell-width};
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: #eee;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+        position: absolute;
+        content: "#{$index}";
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
What did Misha say in his video about setting transition in the :hover? Rewatch that video and fix this